src/io/flutter/inspector/FlutterStudioMonitorStageView.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) 2016 The Android Open Source Project
```

Comment:
are these files with 2016  dates refactors of Android Studio files? If so, adding the original android studio version to the branch history would make reviewing quicker.